File: modul_4/PRAK404-2510817220008-NamiraHumaira.py
while (True):
    print("Pilih program")
    print("1. Pertambahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    pilihan = int(input("Masukkan Pilihan : "))
    if pilihan == 5: break
    if pilihan > 5 or pilihan < 0: 
        print("Input anda salah, silahkan coba lagi")
        continue
    a = float(input("Masukkan nilai pertama : "))
    b = float(input("Masukkan nilai kedua : "))  
    if pilihan == 1: 
        print(f"Hasil penjumlahan antara {a:.2f} dan {b:.2f} adalah {a + b:.2f}")
    elif pilihan == 2 :
        print(f"Hasil pengurangan antara {a:.2f} dan {b:.2f} adalah {a - b:.2f} ")
    elif pilihan == 3 :
        print(f"Hasil perkalian antara {a:.2f} dan {b:.2f} adalah {a * b:.2f} ")
    elif pilihan == 4 :
        if b != 0: print(f"Hasil pembagian antara {a:.2f} dan {b:.2f} adalah {a / b:.2f} ")
        else: print("Hasil pembagian tidak terdefinisi")
print("Terima kasih, telah menggunakan kalkulator NAMIRA HUMAIRA")

File: modul_4/PRAK404-2510817220008-NamiraHumaira.c
#include <stdio.h>
int main()
{
    int pilihan;
    float a, b;
    while(1)
    {
        printf("Pilih program \n");
        printf("1. Penjumlahan \n");
        printf("2. Pengurangan \n");
        printf("3. Perkalian \n");
        printf("4. Pembagian \n");
        printf("5. Exit \n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &pilihan);
        if(pilihan == 5) break; 
        if(pilihan > 5 || pilihan < 1){
            printf("Input anda salah, silahkan coba lagi \n");
            continue;
        }
        printf("Masukkan nilai pertama : ");
        scanf("%f", &a);
        printf("Masukkan nilai kedua : ");
        scanf("%f", &b);
        switch(pilihan)
        {
            case 1:
                printf("Hasil penjumlahan antara %.2f dan %.2f adalah %.2f \n", a, b, a + b);
                break;
            case 2:
                printf("Hasil pengurangan antara %.2f dan %.2f adalah %.2f \n", a, b, a - b);
                break;
            case 3:
                printf("Hasil perkalian antara %.2f dan %.2f adalah %.2f \n", a, b, a * b);
                break;
            case 4:
                if (b != 0) printf("Hasil pembagian antara %.2f dan %.2f adalah %.2f \n", a, b, a / b); 
                else printf("Hasil pembagian tidak terdefinisi \n");
        }
    }
    printf("Terima kasih, telah menggunakan kalkulator NAMIRA HUMAIRA");
    return 0;
}